prac_06/programming_language.py

```diff
@@ -0,0 +1,20 @@
+class ProgrammingLanguage:
+    """Represent a Language object."""
+
+    def __init__(self, name, typing, reflection, year):
+        """Initialise a Language instance."""
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        """Determines if typing is dynamic."""
+        if self.typing == "Dynamic":
```

Comment:
does this need an if else when the condition:
self.typing == "Dynamic" 
will give True or False anyway?